packages/docusaurus-theme-translations/locales/hu/theme-common.json

```
@@ -0,0 +1,70 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "Bezárás",
+  "theme.BackToTopButton.buttonAriaLabel": "Vissza a tetejére",
+  "theme.CodeBlock.copied": "Másolva",
+  "theme.CodeBlock.copy": "Másolás",
+  "theme.CodeBlock.copyButtonAriaLabel": "Másolás a vágólapra",
+  "theme.CodeBlock.wordWrapToggle": "Szótörés váltása",
+  "theme.DocSidebarItem.toggleCollapsedCategoryAriaLabel": "Összecsukás/kibontás kategória '{label}'",
+  "theme.ErrorPageContent.title": "Hiba történt a oldalon.",
+  "theme.ErrorPageContent.tryAgain": "Próbáld újra",
+  "theme.NavBar.navAriaLabel": "Fő",
+  "theme.NotFound.p1": "Sajnos nem találtuk azt az oldalt, amit kerestél.",
+  "theme.NotFound.p2": "Kérjük, lépj kapcsolatba az oldal tulajdonosával, ahonnan erre a hivatkozásra léptél, hogy jelezd, hogy a hivatkozás nem működik.",
+  "theme.NotFound.title": "Az oldal nem található",
+  "theme.TOCCollapsible.toggleButtonLabel": "Ezen az oldalon tartalomjegyzék",
+  "theme.admonition.caution": "vigyázat",
+  "theme.admonition.danger": "veszély",
+  "theme.admonition.info": "információ",
+  "theme.admonition.note": "megjegyzés",
+  "theme.admonition.tip": "tanács",
+  "theme.blog.archive.description": "Archívum",
+  "theme.blog.archive.title": "Archívum",
+  "theme.blog.paginator.navAriaLabel": "Bloglista oldalának navigációja",
+  "theme.blog.paginator.newerEntries": "Újabb bejegyzések",
+  "theme.blog.paginator.olderEntries": "Régebbi bejegyzések",
+  "theme.blog.post.paginator.navAriaLabel": "Blogbejegyzés oldalának navigációja",
+  "theme.blog.post.paginator.newerPost": "Újabb bejegyzés",
+  "theme.blog.post.paginator.olderPost": "Régebbi bejegyzés",
+  "theme.blog.post.plurals": "{count} bejegyzés|{count} bejegyzés|{count} bejegyzés",
+  "theme.blog.post.readMore": "Olvass tovább",
+  "theme.blog.post.readMoreLabel": "További információ erről: {title}",
+  "theme.blog.post.readingTime.plurals": "{readingTime} perc olvasás",
+  "theme.blog.sidebar.navAriaLabel": "Navigáció a blog legutóbbi bejegyzései között",
+  "theme.blog.tagTitle": "{nPosts} a(z) \"{tagName}\" címkével",
+  "theme.colorToggle.ariaLabel": "Váltás a sötét és világos mód között (jelenleg {mode} van beállítva)",
+  "theme.colorToggle.ariaLabel.mode.dark": "Sötét mód",
+  "theme.colorToggle.ariaLabel.mode.light": "Világos mód",
+  "theme.common.editThisPage": "Szerkesztés GitHub-on",
+  "theme.common.headingLinkTitle": "Közvetlen hivatkozás erre: {heading}",
+  "theme.common.skipToMainContent": "Ugrás a fő tartalomhoz",
+  "theme.docs.DocCard.categoryDescription": "{count} elem|{count} elemek|{count} elemei",
```

```suggestion
  "theme.docs.DocCard.categoryDescription": "{count} elemek",
```

Actually this one is not configured as a plural in our theme so it's unnecessary to use a plural form.

We'll likely have to move this key to allow plural forms because "1 elemek" is probably good, but that's a separate problem that requires modifying all the other translations 😅